src/main/java/eu/dissco/orchestration/backend/repository/MappingRepository.java

```diff
@@ -47,28 +48,53 @@ public MappingRecord getMapping(String id) {
         .fetchOne(this::mapToMappingRecord);
   }
 
+  public MappingRecord getMappingOmitDeleted(String id) {
```

Comment:
Yes let's leave this as is for now. Maybe later we need to remove versions for this from the database as well.